tracer/test/test-applications/integrations/Samples.DataStreams.RabbitMQ/Program.cs

```diff
@@ -12,8 +13,10 @@ public static class Program
         private static readonly string Message = nameof(Message);
         private static readonly string Host = Environment.GetEnvironmentVariable("RABBITMQ_HOST") ?? "localhost";
         
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
         {
+            await SampleHelpers.WaitForDiscoveryService();
```

Comment:
Should we extend this to other samples or there's a particular reason it's needed for this sample?